xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
     CPVRRecording(void);
     CPVRRecording(const PVR_RECORDING &recording, unsigned int iClientId);
+
+    // Prevent copy construction, even for CPVRRecording instances and friends.
+    // Note: Only declared, but intentionally not implemented
+    //       to prevent compiler generated copy ctor and to force
+    //       a linker error in case somebody tries to call it.
+    CPVRRecording(const CPVRRecording &tag);
```

Comment:
For what it's worth I think that leaving these unimplemented is the right thing to do. Walls of texts are fine in headers, after all the header is supposed to describe what each function does. The whole idea here is to not accidentally create copies of objects we assume are one and the same and this seems like the perfect safeguard for that.